.evergreen/run-tests.sh

```diff
@@ -140,7 +140,10 @@ if [ -n "$TEST_ENCRYPTION" ]; then
     export PYMONGOCRYPT_LIB
 
     # TODO: Test with 'pip install pymongocrypt'
-    git clone https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
+    git clone https://github.com/blink1073/libmongocrypt.git libmongocrypt_git
+    pushd libmongocrypt_git
+    git checkout PYTHON-3367
+    popd
```

Comment:
No longer needed?

Reply:
It is still needed until we make a new release of pymongocrypt.

Reply:
Fixed